src/lib/compat/cryptojs.ts

```diff
@@ -0,0 +1,148 @@
+/**
+ * This is a compatibility layer for decrypting AES encrypted base64 encoded
+ * strings from CryptoJS.AES.encrypt
+ *
+ * based on crypto-js 3.1.9-1
+ *
+ * The wallet used to depend on crypto-js for en/decryption of the seed, but
+ * then migrated to using the jolocom-lib (which uses crypto-browserify). The
+ * details of the encryption (IV, key derivation) are slightly different, so
+ * previously stored encrypted seeds needed to be re-encrypted.
+ *
+ */
+
+const CryptoJS = {
+  AES: {
+    decrypt: AESDecrypt,
+  },
+  algo: {
+    AES: {
+      keySize: 256/32,
+      ivSize: 128/32
+    }
+  },
+  kdf: {
+    OpenSSL: {
+      execute: OpenSSLKdfExecute
+    }
+  },
+  format: {
+    OpenSSL: {
+      parse: OpenSSLFormatterParse
+    }
+  }
+}
+
+export default CryptoJS
+
+import { randomBytes, createDecipheriv, createHash } from 'crypto'
+import { JolocomLib } from 'jolocom-lib';
+
+/**
+ * This function emulates CryptoJS.format.OpenSSL.parse (cipher-core.js)
+ * It parses a base64 cipher text and separates the salt bytes if any
+ */
+function OpenSSLFormatterParse(cipherTextBase64: string) {
+  let salt, cipherText = Buffer.from(cipherTextBase64, 'base64')
+  // Test for salt
+  if (cipherText.readUInt32BE(0*4) == 0x53616c74 && cipherText.readUInt32BE(1*4) == 0x65645f5f) {
+    // Extract salt, 64 bytes, word  2 and word 3
```

Comment:
64 bits?

Reply:
Also It seems the salt is the 3rd and 4th words (as opposed to word 2 and 3), with the first two words being the hex representation of `Salted__`. Correct me if I'm wrong though.

Reply:
64bits yes!
Word 2 and 3 zero-indexed, that's confusing will fix. And yes about `"Salted__"`! I wasn't aware actually, just checked now :D